src/shared/options/readOptions.ts

```diff
@@ -35,42 +37,38 @@ export async function readOptions(args: string[]): Promise<OptionsParseResult> {
 		tokens: true,
 	});
 
-	const options = {
-		author: values.author as string | undefined,
-		base: values.base as InputBase,
-		createRepository: values["create-repository"] as boolean | undefined,
-		description: values.description as string | undefined,
-		email: values.email as string | undefined,
-		excludeCompliance: values["exclude-compliance"] as boolean | undefined,
-		excludeContributors: values["exclude-contributors"] as boolean | undefined,
-		excludeLintJson: values["exclude-lint-json"] as boolean | undefined,
-		excludeLintKnip: values["exclude-lint-knip"] as boolean | undefined,
-		excludeLintMd: values["exclude-lint-md"] as boolean | undefined,
-		excludeLintPackageJson: values["exclude-lint-package-json"] as
-			| boolean
-			| undefined,
-		excludeLintPackages: values["exclude-lint-packages"] as boolean | undefined,
-		excludeLintPerfectionist: values["exclude-lint-perfectionist"] as
-			| boolean
-			| undefined,
-		excludeLintSpelling: values["exclude-lint-spelling"] as boolean | undefined,
-		excludeLintYml: values["exclude-lint-yml"] as boolean | undefined,
-		excludeReleases: values["exclude-releases"] as boolean | undefined,
-		excludeRenovate: values["exclude-renovate"] as boolean | undefined,
-		excludeTests: values["unit-tests"] as boolean | undefined,
-		funding: values.funding as string | undefined,
-		owner: values.owner as string | undefined,
-		repository: values.repository as string | undefined,
+	const options = optionsFromArgsSchema.parse({
```

Comment:
Not sure how to properly add error handling here, I assume we'd like to use `safeParse` here and conditionally return

```ts
return {
	cancelled: true,
	options,
};
```

The problem with that is our `options` type would contain a lot of union type from `values` - `string | boolean | undefined` :thinking: